gammapy/maps/tests/test_wcs.py

```diff
@@ -221,3 +222,33 @@ def test_region_mask():
 
     mask = geom.region_mask(regions, inside=False)
     assert np.sum(mask) == 8
+
+
+valid_width_inputs = [
+    (10, (10, 10)),
+    (10*u.deg, (10,10)),
+    ((10,), (10, 10)),
+    ([10, 5], (10, 5)),
```

Comment:
Suggest to drop support for Python lists. Many Numpy functions don't for shape arguments. There's drawbacks and no upsides to support more options for users.